OneCommunityGlobal/HighestGoodNetworkApp
# Jiadong replace badges section

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/65985986/300cfd3d-41ba-4a0a-80f7-dd6ce75d5670)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/65985986/cd0822c0-3b69-4227-a64a-ff1f1e1e8acf)

## Related PRS (if any):
This frontend PR is related to the #https://github.com/OneCommunityGlobal/HGNRest/pull/895
…

## Main changes explained:
replace badge section on the dash board to Timlog component
add number indicator
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
4. log as admin user
5. go to dashboard -> Other Links -> Badge Management -> assign a new badge to another user.
6. log out
7. Log in as the user who was recently assigned the badge.
8. Check the number indicator on the dashboard.
9. Click on the "Badge" tab to view the assigned badge.
10. Check if the number indicator reset to 0.

JAE'S ADDITION: IF GOING TO THE BADGES MANAGEMENT PAGE CRASHES THE APP FOR YOU, PLEASE COMMENT TO LET US KNOW AND THEN ADD IT TO THE TOP OF THE BADGES BUGS SECTION ON THE BUGS DOC. THEN COMMENT THERE FOR MY APPROVAL. 

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/65985986/4f18ff23-d1f3-4562-bd42-c5b69542eb6c
